### PR TITLE
Updating the directory structure for the RBUploader component.

### PR DESCRIPTION
**Pull request by weerd**
### What does this PR do?
This PR does a bit of refactoring with the ReportbackUploader component directory to begin moving it towards our newly established directory structure for components. It also alphabetizes the `reducers/index.js` and adds a new `uploads` property to the Redux store initial state object.

_NOTE:_ The code in the `ReportbackUploader/ReportbackUploader.js` file is the same code that used to be in the `ReportbackUploader/index.js`; it just got moved and the index is now a "table of contents" type pattern.

### Any background context you want to provide?
This is the first step towards refactoring/upgrading/simplifying the ReportbackUploader 💃 

### What are the relevant tickets/cards?
Refs https://trello.com/c/CifTBgbt/682-8-as-a-developer-i-want-to-do-some-cleanup-on-the-rb-uploader-and-make-some-enhancements-too

### CODE REVIEW

File: resources/assets/components/ReportbackUploader/ReportbackUploader.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
+import PropTypes from 'prop-types';
+/* global FormData */
```
Comment (DFurnes) [inline resources/assets/components/ReportbackUploader/ReportbackUploader.js]: I know this is just a file rename, but wanna move this bad-boy above the prototypes import? 🔃

File: resources/assets/reducers/index.js [hunk 2]
```diff
@@ -1,12 +1,13 @@
-export campaign from './campaign';
-export reportbacks from './reportbacks';
-export submissions from './submissions';
 export blocks from './blocks';
-export user from './user';
-export signups from './signups';
-export share from './share';
-export events from './events';
-export notifications from './notifications';
+export campaign from './campaign';
```
Comment (DFurnes) [inline resources/assets/reducers/index.js]: 🔡 ✅

File: resources/assets/store.js [hunk 3]
```diff
@@ -64,6 +64,7 @@ const initialState = {
   modal: {
     shouldShowModal: false,
   },
+  uploads: {},
```
Comment (DFurnes) [inline resources/assets/store.js]: Remind me the difference between `uploads` and `submissions`?

Comment (weerd) [inline resources/assets/store.js]: `uploads` is going to provide the state reference that is currently defined internally in the state of the `ReportbackUploader` component. It's like a temp state to house items in the process of getting submitted. 

Just made sense to me to keep it separated and not continue nesting another object inside of submissions. Submissions will now house anything that's been officially submitted (via uploader in current session after clicking submit, or items already submitted and retrieved from PA/Rogue from other sessions).

File: resources/assets/components/ReportbackUploader/ReportbackUploader.js (hunk 1 above)
Comment (weerd) [inline resources/assets/components/ReportbackUploader/ReportbackUploader.js]: sure!

File: resources/assets/store.js (hunk 3 above)
Comment (DFurnes) [inline resources/assets/store.js]: Oh of course! :+1:

**Comment by ngjo** [comment]
[(8) As a developer, I want to do some cleanup on the RB uploader and make some enhancements too!](https://trello.com/c/CifTBgbt/682-8-as-a-developer-i-want-to-do-some-cleanup-on-the-rb-uploader-and-make-some-enhancements-too)

**Pull request merged**